version: "3.4"
services:
  lianmi-authservice:
    image: lianmi/authservice:0.1
    build: ./authservice
    container_name: lianmi-authservice
    restart: unless-stopped
    ports:
      - 28080:28080
    volumes:
      - ${LOG_DIR}/authservice.log:/tmp/authservice.log
    # logging:
    #   driver: fluentd
    #   options:
    #     fluentd-address: localhost:24224
    #     fluentd-retry-wait: '1s'
    #     fluentd-max-retries: '30'
    #     tag: lianmi.lm-cloud.authservice


  lianmi-dispatcher:
    image: lianmi/dispatcher:0.1
    build: ./dispatcher
    container_name: lianmi-dispatcher
    restart: unless-stopped
    volumes:
      - ${LOG_DIR}/dispatcher.log:/tmp/dispatcher.log
    # logging:
    #   driver: fluentd
    #   options:
    #     fluentd-address: localhost:24224
    #     fluentd-retry-wait: '1s'
    #     fluentd-max-retries: '30'
    #     tag: lianmi.lm-cloud.dispatcher


networks:
  default:
    # driver: bridge
    external:
      name: br-lianmi # Create using "docker network create br-lianmi" before run.